sdk/schemaregistry/ci.yml

```diff
@@ -0,0 +1,31 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: template
```

Comment:
this should use schemaregistry rather than template.

Reply:
Fixed.